internals-js/src/specs/listSizeSpec.ts

```diff
@@ -0,0 +1,40 @@
+import { DirectiveLocation } from 'graphql';
+import { createDirectiveSpecification } from '../directiveAndTypeSpecification';
```

Comment:
I'm open to other opinions, but my preference would be for both of these directives to live in the cost spec rather than having a seperate spec for each.

Reply:
I agree, although all my attempts to combine them have failed. When I've combined them into the same spec, I either get some `@listSize directive not known` error or a `duplicate inclusion of feature` error when I try to import both from the same place.

Reply:
Combined to a single spec in https://github.com/apollographql/federation/pull/3074/commits/36d6c71e2c150b6aed0d0bc18362cf31609e7343